### DISCUSSION TITLE
小心釣魚網站

### DISCUSSION THREAD

**Discussion by JoannaLai909** [1101-GS4719-C/Python-Community-GS4719-C#2031] [General]
不久前我在某外國網站消費，該網站採每月訂閱制，但我並未註冊帳號也沒有訂閱過，當時是以訪客身分消費，後來我發現我的信用卡被刷了連續三個月的訂閱金額，詢問銀行客服之後才知道這是釣魚網站盜刷信用卡的手法.....
他們會誘導消費者輸入個人資料和金融卡資訊，藉此他們便有權利對金融卡進行扣款，因為我們是自己輸入的。
後來我上網查了一下，發現釣魚網站不只一種行騙方式，目前已知有通過電子郵件、簡訊以及網站進行釣魚。
而盜刷信用卡只是其中一種後果，更嚴重他們甚至會讓我們的裝置中毒、被勒索或被自動安裝惡意程式等等。
https://www.tsg.com.tw/blog-detail10-299-1-phishing.htm
幸好最後銀行幫我追回被盜刷的錢，也讓我收穫教訓，以後使用電子支付要更加小心，希望大家也要多多留意不要上當!

**Comment by Clare0808** [comment]
在網路消費真的要小心，一不注意都會被騙……我個人如果要到網路買東西，都會選大眾常用的網站，感覺會比較有保障。購買前也都會稍微看一下評論，確定沒問題才下單。

**Comment by peresonacifer** [comment]
感謝你分享這個經驗，也提醒了大家在網絡上進行交易時要保持警惕。檢查網址和安全標誌、避免點擊可疑鏈接、啟用雙重驗證和定期檢查帳單是防範釣魚網站的有效措施。保持設備安全和使用虛擬信用卡也能有效減少被盜刷的風險。

**Comment by CtyceT** [comment]
網路釣魚這麼猖獗必定是現今仍有許多人輕易受騙。例如提醒繳費的簡訊裡的連結真的很容易會讓人不經思考就點進去了，這些訊息和真實官方的訊息實在是長得太像讓人防不慎防。

**Comment by ds0w0** [comment]
感謝你分享這個經驗，也很高興銀行幫你追回了損失。你的經驗提醒我們，網路釣魚詐騙的手法真的挺狡猾的。確認網站的安全性、仔細檢查網址，以及不要隨意點擊不明連結，都是保護自己免受詐騙的重要措施。如果有任何可疑的活動或訊息，及時聯繫銀行或相關機構進行查詢和處理是很重要的。希望未來你能更加安全地使用電子支付(我自己還沒用過電子支付，也希望以後不會遇到這種問題)！

**Comment by hunteryang0108** [comment]
謝謝你的分享，釣魚網站造成的損失確實不能小看。其實還是有一些方法分辨釣魚網站的，例如：安裝防毒軟體的網站篩查功能、仔細辨別網站的網址，這樣做就能大大的降低被釣魚網站欺騙的風險。不過最最重要的還是不要輕易點開陌生的超連結，不要點開就不會有受騙上當的可能，這個方法也可以多多推廣給大家喔。
